certificate-manager/src/main/java/io/strimzi/certs/OpenSslCertManager.java

```diff
@@ -134,18 +134,20 @@ private Path createDefaultConfig() throws IOException {
     /**
      * Add basic constraints and subject alt names section to the provided openssl configuration file
      *
-     * @param sbj subject information
-     *
+     * @param sbj      subject information
+     * @param needsAki if true adds AKI (authorityKeyIdentifier)
      * @return openssl configuration file with subject alt names added
-     *
-     * @throws IOException  Throws IOException when IO operations fail
+     * @throws IOException throws IOException when writing to the file fails
      */
-    private Path buildConfigFile(Subject sbj, boolean isCa) throws IOException {
+    private Path buildConfigFile(Subject sbj, boolean isCa, boolean needsAki) throws IOException {
         Path sna = createDefaultConfig();
         try (BufferedWriter out = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(sna.toFile(), true), StandardCharsets.UTF_8))) {
             if (isCa) {
                 out.append("basicConstraints = critical,CA:true,pathlen:0\n");
             }
+            if (needsAki) {
+                out.append("authorityKeyIdentifier = keyid,issuer\n");
+            }
```

Comment:
why we would need to add a distinction here with `needsAki`. I see you are passing true just for signed certificates but not for the self-signed. AFAIU, in regards to the self-signed CA cert, it will contain the SKI and you can also add the AKI which will point to the SKI itself, so it should work as well.
If this distinction is really not needed you can enable the `authorityKeyIdentifier` within the openssl.conf directly.